lib/shared/util.ts

```diff
@@ -11,4 +16,72 @@ export default class Util {
     static indexOf = indexOf;
     static sortBy = sortBy;
     static format = nodeUtil.format;
+    static request = request;
+    static path = path;
+    static fs = fs;
+    static childProcess = childProcess;
+
+    static getPackageJsonFromSource(url: string) {
+        let content: any;
+
+        return this.request({
+            method: "GET",
+            uri: url,
+            json: true,
+            resolveWithFullResponse: true,
+            headers: this.defaultHeaders
+        })
+            .then((response: any) => {
+                content = response.body;
+                if (content.hasOwnProperty("templateType")) {
+                    return content;
+                }
+            })
+            .catch((error: any) => {
+                return {
```

Comment:
Do we need the catch clause here if we are only to return a *differently* formatted error json (that we need to check for where we intend to use the `getPackageJsonFromSource` method)?

Reply:
If a resource is not found the github api returns: Resource not found without any meaningful info on the resource itself. Since we have a number of templates (15) so far, I think it is a good idea to know exactly which resource has failed!

Reply:
Ok, maybe it would be useful to have this info as a comment.